node/src/effects.rs

```diff
@@ -40,68 +37,20 @@ pub fn effects<S: Service>(store: &mut Store<S>, action: ActionWithMeta) {
             // TODO(binier): create init action and dispatch this there.
             store.dispatch(ExternalSnarkWorkerAction::Start);
 
-            #[cfg(feature = "p2p-libp2p")]
-            {
-                p2p_connection_timeouts(store, &meta);
-                store.dispatch(P2pConnectionOutgoingAction::RandomInit);
-
-                p2p_try_reconnect_disconnected_peers(store);
-                p2p_request_best_tip_if_needed(store);
-                p2p_request_snarks_if_needed(store);
+            p2p_timeout_effects(store, &meta);
 
-                store.dispatch(P2pDiscoveryAction::KademliaBootstrap);
-                store.dispatch(P2pDiscoveryAction::KademliaInit);
-            }
+            p2p_request_best_tip_if_needed(store);
+            // p2p_request_snarks_if_needed(store);
```

Comment:
@akoptelov is this one not working anymore?

Reply:
Let me check